File: apps/home-service-platform-node-js-admin/src/api/admin/AdminWhereInput.ts
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { StringFilter } from "../../util/StringFilter";
import { ReviewWhereUniqueInput } from "../review/ReviewWhereUniqueInput";

export type AdminWhereInput = {
  booking?: BookingWhereUniqueInput;
  email?: StringNullableFilter;
  id?: StringFilter;
  name?: StringNullableFilter;
  permissions?: StringNullableFilter;
  review?: ReviewWhereUniqueInput;
  role?: StringNullableFilter;
};

File: apps/home-service-platform-server/src/APIs/Admin/Dtos/AdminWhereInput.cs
namespace HomeServicePlatform.APIs.Dtos;

public class AdminWhereInput
{
    public DateTime? CreatedAt { get; set; }

    public string? Email { get; set; }

    public string? Id { get; set; }

    public string? Name { get; set; }

    public string? Permissions { get; set; }

    public string? Role { get; set; }

    public DateTime? UpdatedAt { get; set; }
}
